app/views/devices/index.html.erb

```diff
@@ -0,0 +1,84 @@
+<%= render 'shared/partials/nav-global.html' %>
+<main class="devices">
+  <%= render 'shared/partials/header-global.html' %>
+  <nav class="device__selection">
+    <ul>
+      <li class="device__keurig device-selected">
+        <a href="/device-keurig">
+          <%= image_tag("illustrations/ill-keurig.svg", :alt => "Keurig") %>
+          <h3>Keurig</h3>
```

Comment:
@mzlock I believe this shouldn't be an h3, that was my miscalculation, but I do think it should be an `<h2>` like @ccomp suggested. All of the problem statements come under a specific device. To me it makes sense that we would need a heading here.